packages/core/src/plugins/use-masked-errors.ts

```diff
@@ -3,80 +3,67 @@ import { handleStreamOrSingleExecutionResult } from '../utils.js';
 
 export const DEFAULT_ERROR_MESSAGE = 'Unexpected error.';
 
-export type FormatErrorHandler = (error: unknown, message: string, isDev: boolean) => Error;
+export type MaskErrorFn = (error: unknown, message: string) => Error;
+
+export type SerializableGraphQLErrorLike = Error & {
+  name: 'GraphQLError';
+  toJSON(): { message: string };
```

Comment:
I think this should still have extensions as a property in case someone does a console.log 🤔

Reply:
Technically our masking logic doesn't need extensions so it would be extra.